src/dialogs/alwaysonbotDialogs/UpdateProfile/updateProfileDialog.ts

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import {
+    Choice, ChoicePrompt, ComponentDialog, DialogTurnResult, PromptValidatorContext, TextPrompt,
+    WaterfallDialog,
+    WaterfallStepContext
+} from 'botbuilder-dialogs';
+import { CommonPromptValidatorModel } from '../../../models/commonPromptValidatorModel';
+import { ContinueAndFeedbackDialog } from '../Common/continueAndFeedbackDialog';
+import { FeedbackDialog, FeedBack_Dialog } from '../Common/FeedbackDialog';
+import { UpdateAddressDialog, UPDATE_ADDRESS_DIALOG_STEP } from './UpdateAddress/updateAddressDialog';
+import { CommonChoiceCheckDialog, COMMON_CHOICE_CHECK_DIALOG } from './UpdatePhoneNumber/commonChoiceCheckDialog';
+import { UpdateMyPhoneDialog } from './UpdatePhoneNumber/updateMyPhoneDialog';
+
+
+
+const TEXT_PROMPT = 'textPrompt';
+const WATERFALL_DIALOG = 'waterfallDialog';
+const CHOICE_PROMPT = 'CHOICE_PROMPT';
+const UPDATE_PHONE_NUMBER_DIALOG_STEP = 'UPDATE_PHONE_NUMBER_DIALOG_STEP';
```

Comment:
this one should import this  UPDATE_PHONE_NUMBER_DIALOG_STEP from UpdateMyPhoneDialog, not declare one const variable in this file.